app/components/MazemapEmbed/MazemapAccordion.tsx

```diff
@@ -0,0 +1,67 @@
+import { Accordion, Flex, Icon } from '@webkom/lego-bricks';
+import { ChevronRight } from 'lucide-react';
+import { useState } from 'react';
+import mazemapLogo from 'app/assets/mazemap.svg';
+import { FlexRow } from 'app/components/FlexBox';
+import MazemapLink from 'app/components/MazemapEmbed/MazemapLink';
+import { MazemapEmbed } from 'app/components/MazemapEmbed/index';
+import styles from './MazemapAccordion.module.css';
+import type { ComponentProps } from 'react';
+
+type Props = ComponentProps<typeof MazemapEmbed> & {
+  defaultOpen?: boolean;
+};
+
+export const MazemapAccordion = ({ defaultOpen = false, ...props }: Props) => {
+  const [mapIsOpen, setMapIsOpen] = useState(defaultOpen);
+
+  return (
+    <>
+      <Flex column gap="var(--spacing-xs)" className={styles.mazemapAccordion}>
+        <Accordion
+          defaultOpen={defaultOpen}
+          triggerComponent={({ onClick, rotateClassName }) => (
+            <div
+              onClick={() => {
+                onClick();
+                setMapIsOpen(!mapIsOpen);
+              }}
+              className={styles.mazemapAccordionTrigger}
+            >
+              <FlexRow alignItems="center">
```

Comment:
```suggestion
              <FlexRow alignItems="center" gap="var(--spacing-sm)">
```
Could benefit from some minor spacing 🤷🏼‍♂️